src/github.com/getlantern/lantern/feed.go

```diff
@@ -199,10 +184,27 @@ func processFeed(provider FeedProvider) {
 		if desc == "" {
 			desc = entry.Content
 		}
-
 		feed.Entries[i].Description = desc
 	}
 
+	// the 'all' tab contains every article that's not associated with an
```

Comment:
@atavism Ah, I see my mistake now.  I didn't realize that we were using value types here.  If we switch to using pointers, we can avoid some unnecessary copying.  Should I give it a whirl?